docs/docs/recipes.md

```diff
@@ -152,6 +152,51 @@ By targeting the HTML `body` element, your font will apply to most text on the p
 - More on [importing assets into files](/docs/importing-assets-into-files/]
 - [Using Typography.js for Google fonts](/docs/typography-js/)
 
+### Using Google Fonts
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start)
+- The [Gatsby CLI](/docs/gatsby-cli/) installed
+- Choose a font package from [the typefaces project](https://github.com/KyleAMathews/typefaces)
+
+#### Directions
+
+1. Visit [the typefaces project](https://github.com/KyleAMathews/typefaces) and choose the font you'd like.
+2. Run `npm install --save yourchosenfont`, replacing `yourchosenfont` with the name of the font you want to install.
+3. Add `require("yourchosenfont")` to your site's entry file.
+4. Once installed, you can add the font either in your CSS stylesheet or inline using `font-family: yourchosenfont`. An inline implementation might look something like this:
```

Comment:
I'd recommend showing only one method here, and a global stylesheet is the most common.
```suggestion
3. Once it's imported, you can reference the font in a CSS stylesheet.
```